src/API/Site/Controllers/MerchantCenter/AccountController.php

```diff
@@ -76,7 +76,7 @@ public function __construct( ContainerInterface $container ) {
 		parent::__construct( $container->get( RESTServer::class ) );
 		$this->middleware    = $container->get( Middleware::class );
 		$this->merchant      = $container->get( Merchant::class );
-		$this->account_state = $container->get( MerchantAccountState::class );
+		$this->account_state = $container->get( AccountState::class );
```

Comment:
I don't think we should be changing this here (and I assume is the reason why the tests are stating that AccountState is not being managed by the container). Fails at the same location when sending the request.

Reply:
@mikkamp You're right, I was a little over-zealous in my find/replace. I've fixed that.